lib/blog-feed-loader.js

```diff
@@ -0,0 +1,65 @@
+var sanitizeHtml = require('sanitize-html');
```

Comment:
I appreciate the concern for security, but I'm concerned about the weight an entire HTML sanitizer will add to our JS bundle. Do you think you can just use the `feed.entries[].contentSnippet` property of the [JSON result](https://developers.google.com/feed/v1/devguide#resultJson), which doesn't contain markup? Or perhaps use some simple [HTML tag stripping code](http://stackoverflow.com/a/822464) from StackOverflow and set the content using standard JSX text syntax (i.e., not `dangerouslySetInnerHTML`)?

When we switch to getting this from the Teach API, we can potentially make sure that the server sanitizes HTML before sending it to the client. (I was hoping that the Google Feed API would do this for us, but there doesn't seem to be *any* documentation about whether the HTML is sanitized or not, which is quite annoying.)